app/src/main/java/com/eveningoutpost/dexdrip/utils/Preferences.java

```diff
@@ -915,15 +914,15 @@ public void onCreate(Bundle savedInstanceState) {
             addPreferencesFromResource(R.xml.pref_data_source);
 
             addPreferencesFromResource(R.xml.pref_data_sync);
-            setupBarcodeConfigScanner();
-            setupBarcodeShareScanner();
             bindPreferenceSummaryToValue(findPreference("cloud_storage_mongodb_uri"));
             bindPreferenceSummaryToValue(findPreference("cloud_storage_mongodb_collection"));
             bindPreferenceSummaryToValue(findPreference("cloud_storage_mongodb_device_status_collection"));
             bindPreferenceSummaryToValue(findPreference("cloud_storage_api_base"));
 
             addPreferencesFromResource(R.xml.pref_advanced_settings);
             addPreferencesFromResource(R.xml.xdrip_plus_prefs);
+            setupBarcodeConfigScanner();
```

Comment:
Is this code movement necessary and why?

Reply:
I believe I had to move it down below the line that loads the `R.xml.xdrip_plus_prefs`, because I'm essentially moving the "Auto configure" scanner further down the settings list into the "xDrip+ Extra Settings" section.